packages/amplication-code-gen-types/src/plugin-events-params.ts

```diff
@@ -87,7 +87,7 @@ export interface CreateMessageBrokerClientOptionsFactoryParams
 export interface CreateMessageBrokerServiceParams extends EventParams {}
 export interface CreateMessageBrokerServiceBaseParams extends EventParams {}
 export interface CreateServerPackageJsonParams extends EventParams {
-  updateValues: { [key: string]: JsonValue };
+  updateProperties: { [key: string]: any }[];
```

Comment:
just a reminder to update the plugins that use this event

Reply:
https://github.com/amplication/amplication/pull/4131#issuecomment-1275163791